plugin.video.otaku/resources/language/resource.language.pt_br/strings.po

```diff
@@ -78,19 +78,19 @@ msgstr "A autenticação com Real-Debrid falhou, tente novamente"
 
 msgctxt "#30121"
 msgid "Cloudflare Solver"
-msgstr ""
+msgstr "Solucionador Cloudflare"
 
 msgctxt "#30122"
 msgid "Enable Flaresolverr"
-msgstr ""
+msgstr "Ativar o solucionador Flare"
```

Comment:
`Flaresolverr` is the name of the software and should be treated as is and not translated as `flare solver`